src/store/reducers/devicesReducer.js

```diff
@@ -82,6 +82,22 @@ const insertDeviceReducer = (state, { payload }) => {
   });
 };
 
+const updateTagsReducer = (state, { payload }) => {
+  let updatedTagData = {};
+  payload.commonTags.map(({ name, value }) => (updatedTagData[name] = value));
+
+  const updatedDevices = payload.deviceId
+    .map((id) => state.entities[id])
+    .map((device) => update(device, { tags: { $merge: updatedTagData } }))
```

Comment:
You can do all the updates in one go.

```
const updatedDevices = payload.deviceIds
  .map(id => update(getEntities(state)[id], {
    tags: { $merge: updatedTagData, $unset: payload.deletedTags }
  }));
```

Reply:
I kept looking for an example of how to do this!